addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2741,7 +2741,11 @@ export class OdooEditor extends EventTarget {
         };
 
         for (const hint of this.editable.querySelectorAll('.oe-hint')) {
-            if (hint.classList.contains('oe-command-temporary-hint') || !isEmptyBlock(hint)) {
+            if (
+                hint.classList.contains('oe-command-temporary-hint') ||
+                !isEmptyBlock(hint) ||
+                hint.querySelector('T[t-out]')
```

Comment:
It's a bugfix that is somewhat related to this PR, as the bug is much more visible now that the dynamic placeholder generate `<T t-out>` elements.
I can put this bugfix in it's own commit tho, if you prefer.

Reply:
Yes please, move all bug fixes to separate commits. As long as they have the task id it's ok that they are separate from the commit that made the bug more visible. It will be easier for people in the future who might be looking into those fixes, otherwise they might think the changes were strongly related to the feature here while it's not the case. It's just that the feature here makes it way easier to trigger the bug but that's a detail that is going to be mostly irrelevant to whoever is trying to understand the fix, and if they happen to be in the particular case where it is relevant, then they can follow the task id and find the other commits.

Reply:
will add a separate commit with this change